cmd/kyma/completion/cmd.go

```diff
@@ -15,7 +15,46 @@ func NewCmd() *cobra.Command {
 		Use:   "completion bash|zsh",
 		Short: "Generates bash or zsh completion scripts.",
 		Long: `Use this command to display the shell completion code used for interactive command completion. 
-To configure your shell to load completions, add ` + "`. <(kyma completion bash)`" + ` to your bash profile or ` + "`. <(kyma completion zsh)`" + ` to your zsh profile.
+To configure your shell to load completions, use:
+
+Bash:
+
+  $ source <(kyma completion bash)
+
+  # To load completions for each session, execute once:
+  # Linux:
+  $ kyma completion bash > /etc/bash_completion.d/kyma
+  # macOS:
+  $ kyma completion bash > /usr/local/etc/bash_completion.d/kyma
+
+Zsh:
+
+  $ source <(kyma completion zsh)
+
+  # To load completions for each session, execute once:
+  $ kyma completion zsh > "${fpath[1]}/_kyma"
+
+  # You will need to start a new shell for this setup to take effect.
+
+  # If shell completion is not already enabled in your environment,
```

Comment:
I'm not a fan of splitting sentences across two lines, but you probably had reasons? 
Otherwise, I suggest to add the remaining words in this line, and drop them from the next line.

```suggestion
  # If shell completion is not already enabled in your environment, you must enable it.
```